kurtosis-module/impl/module.go

```diff
@@ -214,67 +195,74 @@ func (e Eth2KurtosisModule) Execute(enclaveCtx *enclaves.EnclaveContext, seriali
 		participant_network.ParticipantCLClientType_Teku: teku.NewTekuCLClientLauncher(
 			clGenesisPaths.GetConfigYMLFilepath(),
 			clGenesisPaths.GetGenesisSSZFilepath(),
+			numParticipants,
 		),
 		participant_network.ParticipantCLClientType_Nimbus: nimbus.NewNimbusLauncher(
 			clGenesisPaths.GetParentDirpath(),
```

Comment:
Does Nimbus not need this attribute "numParticipants" such the other CL clients?